salt/etcd/init.sls

```diff
@@ -66,10 +66,6 @@ etcd:
     {%- if should_register_etcd_member %}
       - add-etcd-to-cluster
     {%- endif %}
-  # wait until etcd is actually up and running
-  caasp_etcd.healthy:
```

Comment:
I don't think this should go away. The previous check is just confirming that etcd is running in a stable way in this machine, but this state is making sure the etcd cluster is healthy as a whole, taking all etcd members into account.

Reply:
yop